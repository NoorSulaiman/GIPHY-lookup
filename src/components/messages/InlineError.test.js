import React from 'react';
import { shallow } from 'enzyme';
import InlineError from './InlineError';

it('should render InlineError correctly with given strings', () => {
    const strings = 'error';
    const component = shallow(<InlineError text={strings} />);
    expect(component).toMatchSnapshot();
});