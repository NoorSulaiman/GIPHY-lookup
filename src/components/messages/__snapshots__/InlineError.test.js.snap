// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render InlineError correctly with given strings 1`] = `
<span
  style={
    Object {
      "color": "#ae5856",
    }
  }
>
  error
</span>
`;
